plan/plan.go

```diff
@@ -247,8 +248,12 @@ func (p *Plan) fillAction(ctx context.Context) {
 			a := prevAction.FindByPath(path)
 			if a == nil {
 				v := p.Source().LookupPath(path)
+				name := s.String()
+				if n, err := strconv.Unquote(name); err != nil {
```

Comment:
This should be:
```suggestion
				if n, err := strconv.Unquote(name); err == nil {
```

Reply:
Yes! I've pushed the wrong version initially :)